### DISCUSSION TITLE
支付问题：快到期了，银行卡支付被拒绝，有没有其他的支付方式可以提供？

### DISCUSSION THREAD

**Discussion by kidokogills** [getlantern/lantern-forum-cn#788] [Q&A]
快到期了，剩6天，续费时银行卡被拒绝，显示不支持，确实也没有信用卡，请问有没有其他的支付方式可以提供？例如支付宝或者微信渠道？@Mike777Khoo

**Comment by MaxMKMax** [comment]
@kidokogills 您好. 我马上帮您问问看.

**Reply by MaxMKMax** [reply]
@kidokogills 您好. 请您给我您的账号邮箱. 我们的支付团队回邮件给您支付链接.

**Reply by ADong0103** [reply]
我邮箱46219192@qq.com 能给我一个支付链接么

**Reply by ADong0103** [reply]
我也遇到了同样的问题无法支付，如果可以的话请发一个单独生成的支付链接给我，谢谢

**Reply by MaxMKMax** [reply]
好的. 我们的客服会把链接发给您. 过 1-2 天请查看邮箱

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，邮箱 firekydchan@gmail.com ,感谢

**Reply by MaxMKMax** [reply]
@kidokogills 好的. 我们的客服会您发邮箱. 已经通知他们了.

**Reply by MaxMKMax** [reply]
@ADong0103 好的. 我们的客服会您发邮箱. 已经通知他们了.

**Reply by sobai14150** [reply]
> @kidokogills 您好. 我马上帮您问问看.
你好 购买蓝灯激活码网面失效打不开 之后要怎样才能购买到蓝灯激活码

**Reply by henrylyubj** [reply]
> 好的. 我们的客服会把链接发给您. 过 1-2 天请查看邮箱

我也是这个问题，henrytownbj@gmail.com

**Comment by MaxMKMax** [comment]
QTVV8-WJT88-T843C-D36BT-3QQF2
@kidokogills 您可以用这个码用一个月免费. 送给您的. 谢谢您支持蓝灯!

**Reply by kidokogills** [reply]
这个码，用不了，然后是支付链接要多久才能收到？

**Reply by kidokogills** [reply]
@Mike777Khoo 好几天了，还没收到任何邮件链接

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，请问支付链接什么时候可以发送？

**Reply by MaxMKMax** [reply]
@kidokogills 
您好. 这时可以购买的链接. 您试试看能用吗. 谢谢您支持蓝灯!
https://buy.stripe.com/cN2dTt4ZGar0bUAbIO

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，已付款，firekyd@163.com ,不知道什么时候能收到激活码，麻烦帮我处理一下

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，已通过支付宝购买了一年专业版，邮箱 firekyd@163.com ,请问要多长时间，或者需要什么方式可以获得激活码，请尽快告知，感谢

**Reply by MaxMKMax** [reply]
@kidokogills 您好. 请您提供我支付单据. 我帮您提交问题上去来帮您拿激活码.

**Reply by kidokogills** [reply]
@Mike777Khoo 好的， 邮箱： firekyd@163.com ,单据：
![微信图片_20240829232805](https://github.com/user-attachments/assets/fbfb71d2-e7d7-46c4-837d-8cc1806a61aa)

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，请问处理需要多长时间？ 确实急

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，请问什么时候可以获得激活码？

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，请问什么时候可以获得激活码？

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，已经续费支付一年好几天了，激活码什么时候可以发给我？

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，支付已有一段时间了，请问什么时候可以获得激活码？

**Reply by txwgoogol** [reply]
😂

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，账号续费已有一段时间了，请问什么时候可以获得激活码，就算有情况，也不至于不回复吧，如果发不了邮箱firekyd@163.com，请发到firekydchan@gmail.com

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，请问什么时候可以收到激活码？

**Reply by kidokogills** [reply]
@Mike777Khoo 请问我的激活码什么时候可以处理？

**Reply by kidokogills** [reply]
@Mike777Khoo 你好，请问激活码什么时候可以处理？

**Comment by sobai14150** [comment]
你好  购买蓝灯激活码网面失效打不开 之后要怎样才能购买到蓝灯激活码
